tests/e2e/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM python:2.7-alpine
 RUN apk add --update gcc libffi-dev musl-dev openssl-dev python-dev
 WORKDIR /src
-COPY requirements.txt pipenv.txt /src/
+
+COPY Pipfile pipenv.txt /src/
 RUN pip install -r pipenv.txt
-RUN pipenv install --system -r requirements.txt --skip-lock
+RUN pipenv install --system --skip-lock
```

Comment:
I think we can remove the —skip-lock, as the lock file shouldn’t be present.

Reply:
Fixed